src/SFA.DAS.EmployerAccounts.Jobs/Data/PopulateRepository.cs

```diff
@@ -0,0 +1,64 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Data;
+using System.Linq;
+using System.Threading.Tasks;
+using Dapper;
+using Microsoft.Extensions.Logging;
+using SFA.DAS.EmployerAccounts.Data;
+
+namespace SFA.DAS.EmployerAccounts.Jobs.Data
+{
+    public class PopulateRepository : IPopulateRepository
```

Comment:
What do you think of having [one repository per aggregate](https://docs.microsoft.com/en-us/dotnet/standard/microservices-architecture/microservice-ddd-cqrs-patterns/infrastructure-persistence-layer-design)?
So perhaps a [User|MembershipUser]Repository (or similar) and a JobHistoryRepository?
Also, I think it would be better if the repository was named after the entities involved, rather than a verb.

Reply:
Yes i see a lot of value in that. But the question then is where does this repository live. It's sole purpose is to populate the ReadStore

Reply:
Have moved the IJobHistoryRepo into the application layer